src/components/PaneRight.vue

```diff
@@ -77,7 +79,7 @@ export default {
 .right-pane-tabs::-webkit-scrollbar {
   display: none;
 }
-.right-pane-tabs,
+.right-pane-tabs-container,
 .right-pane-contexts {
   padding-right: 1.5rem;
```

Comment:
```suggestion
   margin-right: 1.5rem;
```

Reply:
Well, it was quite simple!
Thanks @ydcjeff !